### PR TITLE
test

### PR DESCRIPTION
**Pull request by ferretdayo**

### CODE REVIEW

File: backend/src/main/java/com/example/web/controller/RouteController.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.example.web.entity.EventEntity;
```
Comment (sudadayo) [inline backend/src/main/java/com/example/web/controller/RouteController.java]: MINOR Remove this unused import 'com.example.web.entity.EventEntity'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: backend/src/main/java/com/example/web/controller/RouteController.java [hunk 2]
```diff
@@ -132,7 +133,7 @@ public Object index(@PathVariable("eventId") String eventId, @RequestParam("depa
 		List<Object> list = new ArrayList<>();
 		for(int i = 0; i < navigation.size(); i++) {
 			NavigationEntity data = navigation.get(i);
-			Map<String, Object> entity = new HashMap<>();
+			Map<String, Object> entity = new HashMap<String, Object>();
```
Comment (sudadayo) [inline backend/src/main/java/com/example/web/controller/RouteController.java]: MINOR Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)

File: backend/src/main/java/com/example/web/controller/RouteController.java (hunk 1 above)
Comment (ferretdayo) [reply backend/src/main/java/com/example/web/controller/RouteController.java]: 先生！！すいません！消します＞＜

File: backend/src/main/java/com/example/web/controller/RouteController.java (hunk 2 above)
Comment (ferretdayo) [reply backend/src/main/java/com/example/web/controller/RouteController.java]: Map<String, Object> entity = new HashMap<String, Object>();
を
Map<String, Object> entity = new HashMap<>();
に直します！

**Pull request closed without merging**